WordleGame/WordleGame/Base.lproj/Main.storyboard

```diff
@@ -16,23 +16,22 @@
                         <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
```

Comment:
You can drop the `trailing` constraint for `Keyboard Container` because you already have the `leading` and `centerHorizontally` constraint, the `trailing` constraint is absolute in this case

I suggest you to drop the `top` constraint as well. You have constant 577, it will break the layout on small devices. You can just drop it. The storyboard is highlighted with red but the errors will go away once you add subviews in the code. So you can ignore these errors